### PR TITLE
Update for get started samples

### PR DESCRIPTION
**Pull request by slyubimt**
### Details:
 - Update for commands (omz tools and mo)
 - Remove any reference on 'Security Barrier Camera Demo'  demo.
 - replace squeezenet1.1 with googlenet-v1
 - fix for paths

### CODE REVIEW

**Review by vladimir-dudnik — APPROVED** [review]

File: docs/get_started/get_started_demos.md [hunk 1]
```diff
@@ -10,9 +10,8 @@ You will perform the following steps:
 3. <a href="download-media">Download media files to run inference on.</a>
 4. <a href="run-image-classification">Run inference on the sample and see the results:</a>
     - <a href="run-image-classification">Image Classification Code Sample</a>
-    - <a href="run-security-barrier">Security Barrier Camera Demo application</a>
 
-If you installed OpenVINO™ via `pip` you need to change commands listed below. Details are listed in one of [tutorials](https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/002-openvino-api/002-openvino-api.ipynb).
+If you installed OpenVINO™ via `pip` you can quickly getting started with the product by trying these [tutorials](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks).
```
Comment (xu-yuan1) [inline docs/get_started/get_started_demos.md:14]:
```suggestion
If you installed OpenVINO™ via `pip`, you can quickly getting started with the product by trying these [tutorials](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks).
```

File: docs/get_started/get_started_demos.md [hunk 2]
```diff
@@ -31,6 +30,13 @@ This guide assumes you completed all installation and configuration steps. If yo
   
 @endsphinxdirective
 
+## Install OpenVINO Development Tools
+
+To install OpenVINO Development Tools for working with Caffe* models use following command:
```
Comment (xu-yuan1) [inline docs/get_started/get_started_demos.md:35]:
```suggestion
To install OpenVINO Development Tools for working with Caffe* models, use following command: 
```

File: docs/get_started/get_started_demos.md [hunk 3]
```diff
@@ -122,20 +125,20 @@ Use the Model Downloader to download the models to a models directory. This guid
    Don't run downloader with `sudo`. It will further lead to complications
    .. code-block:: sh
```
Comment (xu-yuan1) [inline docs/get_started/get_started_demos.md:126]:
```suggestion

   .. code-block:: sh
```

File: docs/get_started/get_started_demos.md [hunk 4]
```diff
@@ -122,20 +125,20 @@ Use the Model Downloader to download the models to a models directory. This guid
...
 
    .. code-block:: bat
 
-      python downloader.py --name <model_name> --output_dir <models_dir>
+      omz_downloader --name <model_name> --output_dir <models_dir>
 
 .. tab:: macOS
 
    Don't run downloader with `sudo`. It will further lead to complications
    .. code-block:: sh
```
Comment (xu-yuan1) [inline docs/get_started/get_started_demos.md:139]:
```suggestion

   .. code-block:: sh
```

File: docs/get_started/get_started_demos.md [hunk 5]
```diff
@@ -90,29 +96,26 @@ This guide uses the OpenVINO™ Model Downloader to get pre-trained models. You
...
 
-      cd /opt/intel/openvino_2021/deployment_tools/tools/model_downloader/
-      python3 info_dumper.py --print_all
+      info_dumper --print_all
 
 @endsphinxdirective
 
-### Use `grep` to list models that have a specific name pattern
+### Use 'grep' to list models that have a specific name pattern
```
Comment (xu-yuan1) [inline docs/get_started/get_started_demos.md:115]:
```suggestion
### Use grep to List Models of a Same Naming Pattern
```

You can use the following command to list models of a same naming pattern:

File: docs/get_started/get_started_demos.md [hunk 6]
```diff
@@ -90,29 +96,26 @@ This guide uses the OpenVINO™ Model Downloader to get pre-trained models. You
...
 
 ``` sh
-   python3 info_dumper.py --print_all | grep <model_name>
+   info_dumper --print_all | grep <model_name>
 ```
 
 Use the Model Downloader to download the models to a models directory. This guide uses `<models_dir>` and `<models_name>` as placeholders for the models directory and model name:
```
Comment (xu-yuan1) [inline docs/get_started/get_started_demos.md:121]:
```suggestion
### Use Model Downloader to Download Models to a Directory

This guide uses `<models_dir>` and `<models_name>` as placeholders for the models directory and model name:
```

File: docs/get_started/get_started_demos.md (hunk 5 above)
Comment (xu-yuan1) [reply docs/get_started/get_started_demos.md:115]: I think this heading and the heading above "List the models available in the downloader" should not be headings. They seem to be two options to find models. and the text after them is another topic which should have a heading. Please check.

**Review by ilya-lavrenov — APPROVED** [review]

File: docs/get_started/get_started_demos.md [hunk 7]
```diff
@@ -90,29 +96,26 @@ This guide uses the OpenVINO™ Model Downloader to get pre-trained models. You
...
 
 .. tab:: macOS
 
    .. code-block:: sh
 
-      cd /opt/intel/openvino_2021/deployment_tools/tools/model_downloader/
-      python3 info_dumper.py --print_all
+      info_dumper --print_all
 
 @endsphinxdirective
```
Comment (ilya-lavrenov) [inline docs/get_started/get_started_demos.md:113]: BTW, do we need tabs for OSes? I suppose python tools work in unified manner for al operation systems and we can provide a single command.

Comment (ilya-lavrenov) [reply docs/get_started/get_started_demos.md:113]: the same is valid for other places as well, which are OS-independent.

File: docs/get_started/get_started_demos.md [hunk 8]
```diff
@@ -122,20 +125,20 @@ Use the Model Downloader to download the models to a models directory. This guid
...
 
 .. tab:: macOS
 
    Don't run downloader with `sudo`. It will further lead to complications
    .. code-block:: sh
 
-      python3 downloader.py --name <model_name> --output_dir <models_dir>
+      omz_downloader --name <model_name> --output_dir <models_dir>
 
 @endsphinxdirective
```
Comment (ilya-lavrenov) [inline docs/get_started/get_started_demos.md:144]: the line below still contains reference to "Security Barrier Camera"

File: docs/get_started/get_started_demos.md [hunk 9]
```diff
@@ -460,22 +334,16 @@ As an alternative, the Intel® Distribution of OpenVINO™ toolkit includes seve
 @sphinxdirective
 .. tab:: Linux
```
Comment (ilya-lavrenov) [inline docs/get_started/get_started_demos.md:335]: these tabs seem to be OS-independent

File: docs/get_started/get_started_demos.md [hunk 10]
```diff
@@ -561,27 +429,27 @@ To run the **Image Classification** code sample with an input image using the IR
 
 @endsphinxdirective
 
-The following commands run the Image Classification Code Sample using the `car.png` file from the `demo` directory as an input image, the model in IR format from the `ir` directory, and on different hardware devices:
+The following commands run the Image Classification Code Sample using the [`car.png`](https://storage.openvinotoolkit.org/data/test_data/images/car.png) file as an input image, the model in IR format from the `ir` directory, and on different hardware devices:
```
Comment (ilya-lavrenov) [inline docs/get_started/get_started_demos.md:432]: it's not rendered well for me..

File: docs/get_started/get_started_demos.md (hunk 6 above)
Comment (slyubimt) [reply docs/get_started/get_started_demos.md]: reworked.

File: docs/get_started/get_started_demos.md (hunk 5 above)
Comment (slyubimt) [reply docs/get_started/get_started_demos.md]: reworked

File: docs/get_started/get_started_demos.md (hunk 1 above)
Comment (slyubimt) [reply docs/get_started/get_started_demos.md]: reworked

**Comment by ilya-lavrenov** [comment]
please, port to 2022.1 once the PR is merged

File: docs/get_started/get_started_demos.md [hunk 11]
```diff
@@ -412,9 +411,7 @@ The following commands run the Image Classification Code Sample using the [`car.
 @endsphinxdirective
 
    **MYRIAD:**
-   > **NOTE**: Running inference on VPU devices (Intel® Movidius™ Neural Compute
-   Stick or Intel® Neural Compute Stick 2) with the MYRIAD plugin requires 
-    [additional hardware configuration steps](inference-engine/README.md#optional-additional-installation-steps-for-the-intel-movidius-neural-compute-stick-and-neural-compute-stick-2), as described earlier on this page.
+   > **NOTE**: Running inference on VPU devices (Intel® Movidius™ Neural Compute Stick or Intel® Neural Compute Stick 2) with the MYRIAD plugin requires [additional hardware configuration steps](inference-engine/README.md#optional-additional-installation-steps-for-the-intel-movidius-neural-compute-stick-and-neural-compute-stick-2), as described earlier on this page.
```
Comment (xu-yuan1) [inline docs/get_started/get_started_demos.md:414]:
```suggestion
   > **NOTE**: Running inference on VPU devices (Intel® Movidius™ Neural Compute Stick or Intel® Neural Compute Stick 2) with the MYRIAD plugin requires [additional hardware configuration steps](../install_guides/configurations-for-ncs2.md), as described earlier on this page.
```

**Review by antonrom23 — APPROVED** [review]

File: docs/get_started/get_started_demos.md (hunk 10 above)
Comment (slyubimt) [reply docs/get_started/get_started_demos.md]: replace squeezenet1.1 with googlenet-v1. Use https://storage.openvinotoolkit.org/data/test_data/images/224x224/dog.bmp for classification

**Pull request merged**